hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,10 @@
+RELEASE_TYPE: patch
+
+This patch improves the behaviour of the :func:`~hypothesis.strategies.text`
+strategy when passed an ``alphabet`` which is not a strategy.  The value is
+now interpreted as ``whitelist_characters`` to :func:`~hypothesis.strategies.characters`
+instead of a sequence for :func:`~hypothesis.strategies.sampled_from`, which
+standardises the distribution of examples and the shrinking behaviour.
```

Comment:
Given that the main change here is that it's supposed to standardised this, it would be good to see a test that a nonstandard alphabet order still shrinks in the normal direction.